src/Umbraco.StorageProviders.AzureBlob/AzureBlobExtensions.cs

```diff
@@ -263,7 +263,7 @@ public static IUmbracoBuilder AddCdnMediaUrlProvider(this IUmbracoBuilder builde
         /// <param name="builder">The <see cref="IUmbracoMiddlewareBuilder"/>.</param>
         /// <returns>The <see cref="IUmbracoMiddlewareBuilder"/>.</returns>
         /// <exception cref="ArgumentNullException">Thrown if <paramref name="builder"/> or is null.</exception>
-        public static IUmbracoMiddlewareBuilder WithAzureBlobMediaFileSystem(this IUmbracoMiddlewareBuilder builder)
+        public static IUmbracoApplicationBuilderContext WithAzureBlobMediaFileSystem(this IUmbracoApplicationBuilderContext builder)
```

Comment:
I think we should update the name (from `With` to `Use`) to align with the name change in Core (https://github.com/umbraco/Umbraco-CMS/commit/ab66a9f33c5a1950e81e1b09262b1e4c815efa49)

```suggestion
        public static IUmbracoApplicationBuilderContext UseAzureBlobMediaFileSystem(this IUmbracoApplicationBuilderContext builder)
```